objects/src/accounts/delta/storage.rs

```diff
@@ -96,6 +113,13 @@ impl Serializable for AccountStorageDelta {
             idx.write_into(target);
             value.write_into(target);
         }
+
+        assert!(self.updated_maps.len() <= u8::MAX as usize, "too many updated storage maps");
+        target.write_u8(self.updated_maps.len() as u8);
+        for (idx, value) in self.updated_maps.iter() {
+            idx.write_into(target);
+            value.write_into(target);
+        }
```

Comment:
**Request**: change the serialization/deserialization to use the generic vec implementation.